Sources/StreamChatSwiftUI/ChatChannel/Reactions/MessageActions/DefaultMessageActions.swift

```diff
@@ -224,4 +249,74 @@ extension MessageAction {
         
         return flagMessage
     }
+    
+    private static func muteAction(
+        for message: ChatMessage,
+        channel: ChatChannel,
+        chatClient: ChatClient,
+        mutedUser: ChatUser,
```

Comment:
That's pretty pedantic, but technically a better name would be in the realm of `userToMute` as at this point the user is not yet muted.

Could lead to confusion sometime in the future.